e2e/src/main/kotlin/org/jetbrains/bsp/bazel/BazelBspScalaProjectTest.kt

```diff
@@ -0,0 +1,48 @@
+package org.jetbrains.bsp.bazel
+
+import ch.epfl.scala.bsp4j.BuildTarget
+import ch.epfl.scala.bsp4j.BuildTargetCapabilities
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier
+import ch.epfl.scala.bsp4j.WorkspaceBuildTargetsResult
+import ch.epfl.scala.bsp4j.PythonBuildTarget
+import ch.epfl.scala.bsp4j.BuildTargetDataKind
+import ch.epfl.scala.bsp4j.ScalaBuildTarget
+import ch.epfl.scala.bsp4j.ScalacOptionsItem
+import ch.epfl.scala.bsp4j.ScalacOptionsParams
+import ch.epfl.scala.bsp4j.ScalacOptionsResult
+import org.jetbrains.bsp.bazel.base.BazelBspTestBaseScenario
+import org.jetbrains.bsp.bazel.base.BazelBspTestScenarioStep
+
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Duration.Companion.seconds
+
+object BazelBspScalaProjectTest : BazelBspTestBaseScenario() {
```

Comment:
Trying to make this work currently, but getting:
```
ERROR: error loading package '': Every .bzl file must have a corresponding package, but '//utils:tools.bzl' does not have one. Please create a BUILD file in the same or any parent directory. Note that this BUILD file does not need to do anything except exist.
```
and it has the BUILD file

Reply:
did you try locking the version for Bazel and test locally (by running this as a standalone Bazel project)?

Reply:
Looks like files are not copied correctly, at some point it stopped finding WORKSPACE file as well :/

Reply:
@abrams27 maybe you know about this issue?

Reply:
It seems that only the main directory is copied, any subdirectories are not :/ I don't really understand how is that possible. Works correctly for the python one on the same machine

Reply:
Figured it out, it needs to be added to https://github.com/JetBrains/bazel-bsp/blob/master/.bazelrc#L9 <- any chance to fix that requirement?

Reply:
Noted.
I think we can later write detailed instructions for adding a new e2e test for this repo.
Or auto-generate `.bazelrc` to cater for the tests.
We will see about that.